pbft/pbft.cpp

```diff
@@ -1163,6 +1722,53 @@ pbft::already_seen_request(const bzn_envelope& req, const request_hash_t& hash)
     return false;
 }
 
+/*!
+ * Creates <VIEW-CHANGE v+1, n, C, P, i>_sigma_i
+ * @param new_view new view (v+1)
+ * @param base_sequence_number sequence of last stable checkpoint (n)
+ * @param stable_checkpoint_proof proof of checkpoint C
+ * @param prepared_operations set of messages after i was prepared
+ * @param i the uuid of the sender
+ *
+ * @return <VIEW-CHANGE v+1, n, C, P, i>_sigma_i
+ */
+pbft_msg
+pbft::make_viewchange(
+        uint64_t new_view
+        , uint64_t base_sequence_number
+        , std::unordered_map<bzn::uuid_t, std::string> stable_checkpoint_proof
+        , std::unordered_set<std::shared_ptr<bzn::pbft_operation>> prepared_operations)
+{
```

Comment:
parameters should be references to avoid copying the map/set

Reply:
Done. Also made them const.